droidlet/tools/crowdsourcing/droidlet_static_html_task/issue_bonus.py

```diff
@@ -17,49 +20,85 @@
 db = LocalMephistoDB()
 data_browser = DataBrowser(db=db)
 
-def main(run_id) -> list:
-    logging.info(f"Initializing bonus script for Mephisto job {run_id}")
+s3 = boto3.client('s3')
+
+
+def issue_bonuses(task_name: str) -> list:
+    logging.info(f"Initializing bonus script for Mephisto task_name: {task_name}")
+
+    logging.info(f"Downloading interaction bonus records from S3...")
+    with open("bonus_records.csv", "wb") as f:
+        s3.download_fileobj('droidlet-hitl', 'bonus_records.csv', f)
+
+    logging.info(f"Building list of already issued bonuses...")
+    issued_units = []
+    with open('bonus_records.csv', newline='') as csvfile:
+        reader = csv.reader(csvfile)
+        for row in reader:
+            issued_units.append(row[1])  # unit_id should be unique to each interaction HIT
```

Comment:
unit_id & run_id are local to your mephisto database. There could be unit_id conflicts if multiple people are updating `bonus_records.csv` file. I think we should use at least 'batch_id + unit_id' to identify individual units.